src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -725,44 +855,190 @@ function wait_for_dynamo_frontend()
+    -l 5 \
+    -f ${RESULTS_DIR}/gpu_utilization-${SLURM_NODEID}.csv
+}
+
+function launch_workload()
+{
+  local workload_config_name="$1"
+  local workload_args_name="$2"
+
+  # Create nameref to the associative arrays
+  local -n workload_config_ref="$workload_config_name"
+  local -n workload_args_ref="$workload_args_name"
```

Shellcheck (0.11.0)

[warning] workload_args_ref appears unused. Verify use (or export if used externally).

(SC2034)